### DISCUSSION TITLE
Where's the MapControl documentation?

### DISCUSSION THREAD

**Discussion by tipa** [microsoft/microsoft-ui-xaml#9455] [Q&A]
I might be missing the obvious but I don't find any onboarding on how to use the new MapControl.
Is there any documentation? How do I get a MapServiceToken? What methods are available?
I've googled for half an hour and didn't find any starting point

**Comment by brabebhin** [comment]
Winui controls are the same as uwp, the documentation works across both of them.

**Reply by tipa** [reply]
> Winui controls are the same as uwp

I wish....

**Reply by ayamadori** [reply]
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.mapcontrol?view=windows-app-sdk-1.5
